ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -527,7 +531,12 @@ export class WalletDB {
   }
 
   async getUnconfirmedBalance(account: Account, tx?: IDatabaseTransaction): Promise<bigint> {
-    const unconfirmedBalance = await this.balances.get(account.id, tx)
+    // TODO(mgeist,rohanjadvani): This will be updated in a subsequent PR to
+    // include an asset as an argument
+    const unconfirmedBalance = await this.balances.get(
+      [account.prefix, Asset.nativeIdentifier()],
+      tx,
+    )
     Assert.isNotUndefined(unconfirmedBalance)
```

Comment:
Wondering if it makes sense to return 0 if there is no balance stored for the asset instead of asserting here

Reply:
Yep, I am going to do that in a subsequent PR. This is stubbed out to just include the native identifier right now. When I add `assetIdentifier` added as an argument to this method, that logic will be updated to return `BigInt(0)` if no record is found.